Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# 3rd Bug Crusade

## Changelog:
- Fixed an issue where Lordaeron could build a Grand Mosque instead of Grand Cathedral.


## Developer changelog:
- Replaced `intellect_good_1` by `shrewd`.


# How to test:
Check the bug mentioned in the changelog is no more.
If you can, check the Files changed on Github.
